langchain/callbacks/aim_callback.py

```diff
@@ -269,7 +269,7 @@ def on_chain_start(
         inputs_res = deepcopy(inputs)
 
         self._run.track(
-            aim.Text(inputs_res["input"]), name="on_chain_start", context=resp
+            aim.Text(inputs_res["query"]), name="on_chain_start", context=resp
```

Comment:
unless i'm misunderstanding, isn't this dependent on the specific chain? not sure how we hardcoded this in first place

Reply:
@dev2049 @agola11 Gotcha, didn't know that. Made another commit to be key agnostic.

Reply:
Hello, how about calling the first item of the dictionary? @mcleonard @tmynn @dev2049 @agola11 
```
aim.Text(list(inputs_res.values())[0]), name="on_chain_start", context=resp
```

For `search` chain, **inputs_res** has 'input' key.  However, for `math` chain, **inputs_res** has only 'question' key, as you find below. 
```
> Entering new  chain...
{'input': "Who is Leo DiCaprio's girlfriend? What is her current age raised to the 0.43 power?"}
 I need to find out who Leo DiCaprio's girlfriend is and then calculate her age raised to the 0.43 power.
Action: Search
Action Input: "Leo DiCaprio girlfriend"
Observation: The Oscar-winner was then linked to Gigi Hadid, 28, in September 2022.
Thought: I need to calculate Gigi Hadid's age raised to the 0.43 power.
Action: Calculator
Action Input: 28^0.43

> Entering new  chain...
{'question': '28^0.43'}
Error in on_chain_start callback: 'input'
28^0.43
\```text
28**0.43
\```
...numexpr.evaluate("28**0.43")...

Answer: 4.1906168361987195
> Finished chain.
Error in on_chain_end callback: 'output'

Observation: Answer: 4.1906168361987195
Thought: I now know the final answer.
Final Answer: Gigi Hadid, 28, is Leo DiCaprio's girlfriend and her age raised to the 0.43 power is 4.1906168361987195.

> Finished chain.
```

Same solution goes for `on_chain_end`.
Looking forward fixing these trivial errors.